### ISSUE TITLE
Tox ID isn't displayed

### ISSUE THREAD

**Issue by redmanmale** [uTox/uTox#592]
Tox ID field looks like this before selection:
![](https://puu.sh/t1Gji/760be89c83.png)
after selection:
![](https://puu.sh/t1Gka/3f722d981a.png)

But ID could not be copied either via CTRL-C or button COPY.

Win 7, 8.1 x64, uTox v.0.11.1

**Comment by robinlinden** [comment]
Has this profile been used in qTox too?

**Comment by redmanmale** [comment]
No. Never. Only uTox since the very old versions.
Confirmed on two PCs. And only with 0.11.1, old 0.9.7 was just fine.

**Comment by redmanmale** [comment]
Hm. After couple restarts on Win 8.1 this problem was gone.

**Comment by robinlinden** [comment]
That's really strange. We've had this reported before (and I could recreate it) on profiles created in qTox. Glad it started working again though. I'll see if I can find anything.

**Comment by redmanmale** [comment]
OK, thanks.
I'll check it with another uTox profile on Win 7 when I can get access to it.

**Comment by GrayHatter** [comment]
Yeah, there's a race condition I know about if toxcore is able to start before the gui finishing it's init.

**Comment by Appius** [comment]
The same problem here. Windows 10 x64, uTox 0.9.7.
Have never used qTox or any other clients.

![2017-01-03](https://cloud.githubusercontent.com/assets/5452634/21601296/b67f6fe8-d180-11e6-8c64-96a0c1c4a66d.png)

**Comment by GrayHatter** [comment]
this should be fixed in develop via better tox core init from @cebe

**Comment by robinlinden** [comment]
Can still recreate on [65ab802](https://github.com/uTox/uTox/commit/65ab802962d2469430636de64576ea170870dac2)
![No Tox ID](https://cloud.githubusercontent.com/assets/8304462/22262942/0b0268c4-e273-11e6-8618-553f7cc76256.png)

**Comment by redmanmale** [comment]
Still here in v.0.12.2

**Comment by redmanmale** [comment]
And in v.0.14.0 too.

**Comment by GrayHatter** [comment]
does the copy button work?

**Comment by redmanmale** [comment]
I've been trying to reproduce this error for a couple of days, yet no luck here. I'll report, if (when) I finally catch it.

**Comment by redmanmale** [comment]
Got it!
Button doesn't work.

**Comment by redmanmale** [comment]
Still here in 0.15.0.

**Comment by redmanmale** [comment]
0.16.1
